README.md

```diff
@@ -82,10 +82,10 @@ vendor
 
 ### Installing the Dependencies
 
-Fulfilling the `arduino_ci` library dependency is as easy as running this command:
-
+Fulfilling the `arduino_ci` library dependency is as easy as running either of these two commands:
 ```
-$ bundle install
+$ bundle install   # adds packages to global library (may require admin rights)
+$ bundle install --path vendor/bundle   # adds packages to local library
```

Comment:
Did you ever attempt `bundle install --deployment` ?  I'm curious if that worked

Reply:
Yes, and it did not work. I reported the same error as a simple install (asking for a password and suggesting the local path as an alternative).